### PR TITLE
Navbar minecard

### PR DESCRIPTION
**Pull request by harsh-quadB**
## Description
Please include a summary of the changes and the purpose of this PR.

- What does this PR do?
- Why is it needed?

## Screenshots
Include screenshots for **both desktop and mobile** versions.

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Briefly explain the approach taken to implement the changes. 

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/game_frontend/src/components/ui/Navbar/index.css [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+
 .top-nav-bar {
-    @apply absolute top-[72px] left-[64px] right-[64px] w-[1400px] h-[72px] flex items-center justify-between bg-[#D9CCC1] opacity-60 p-2;
+    @apply  top-[72px] left-[64px] right-[64px] max-w-[900px] w-auto mx-auto h-[72px] flex items-center justify-between bg-[#D9CCC1] opacity-60 p-2;
   }
   
   /* Content wrapper */
   .nav-content {
-    @apply flex items-center w-full h-full space-x-2;
+    @apply flex items-center w-full h-full gap-x-2;
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/ui/Navbar/index.css:9]: It should be `gap-2` not `gap-x-2`.

File: src/game_frontend/src/components/ui/Navbar/index.css [hunk 2]
```diff
@@ -1,18 +1,19 @@
-/* Main navbar container */
+/* Main navbar container chNG */
+
 .top-nav-bar {
-    @apply absolute top-[72px] left-[64px] right-[64px] w-[1400px] h-[72px] flex items-center justify-between bg-[#D9CCC1] opacity-60 p-2;
+    @apply  top-[72px] left-[64px] right-[64px] max-w-[900px] w-auto mx-auto h-[72px] flex items-center justify-between bg-[#D9CCC1] opacity-60 p-2;
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/ui/Navbar/index.css:4]: Remove the whole top, left, right styles cause they are not applicable for containers with `position : relative`

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

**Review by ArjunQBTech — COMMENTED** [review]
Great

**Review by ArjunQBTech — APPROVED** [review]
Great! Looks good to me.

**Comment by harsh-quadB** [comment]
remove top, left, and right properties and also update gap-x-2 Please check it once

**Pull request merged**